.github/workflows/docker_images.yml

```diff
@@ -60,6 +82,11 @@ jobs:
         docker push pdidevel/xenial_libs_cmake3.5
         docker push pdidevel/xenial_libs_cmake3.10
         docker push pdidevel/xenial_pdi
+        docker push pdidevel/xenial_cmake3.5:${image_version}
```

Comment:
Doesn't `docker push pdidevel/xenial_cmake3.5` push all the versions of the tag?

Reply:
Given the help of it, I guess that's the case when you add `-a` only

```
Usage:  docker push [OPTIONS] NAME[:TAG]

Push an image or a repository to a registry

Options:
  -a, --all-tags                Push all tagged images in the repository
      --disable-content-trust   Skip image signing (default true)
  -q, --quiet                   Suppress verbose output
```